log.go

```diff
@@ -0,0 +1,46 @@
+package authenticated_block_store
+
+import (
+	"fmt"
+)
+
+type LogManager struct{
+        fileLimit int
+}
+
+// Table API, called by: package.api()
+
+func NewLogManager() (*LogManager, error) {
+	var newLogMgr *LogManager
+	return newLogMgr, nil
+}
+
+// LogManager interfaces, called by: receiver.method()
```

Comment:
No need to document how to call a method on an object, you can assume the reader understands that.